static/app/components/core/alert/index.chonk.tsx

```diff
@@ -95,6 +106,17 @@ function makeChonkAlertTheme(
   throw new TypeError(`Invalid alert type, got ${type}`);
 }
 
+function generateAlertBackground(
+  props: ChonkAlertProps,
+  tokens: ReturnType<typeof getChonkAlertTokens>
+) {
+  const width = 44;
+  if (props.showIcon) {
+    return `linear-gradient(to right, ${tokens.iconBackground}, ${tokens.iconBackground} ${width - 1}px, ${tokens.iconBackground} ${width - 1}px, ${tokens.border} ${width - 1}px, ${tokens.border} ${width}px, ${tokens.background} ${width}px, ${tokens.background} ${width + 1}px)`;
```

Comment:
Can we use `css` template here to get this to auto format.

Reply:
Great suggestion, addressed in 94b6bcb8e4f